tests/qos/test_qos_sai.py

```diff
@@ -884,6 +885,11 @@ def testQosSaiDwrr(
             "hwsku":dutTestParams['hwsku'],
             "topo": dutTestParams["topo"]
         })
+        if qos_remap_enable:
```

Comment:
we don't need this check. we can always update testParams with both the values

Reply:
Good point. Fixed.